etl/steps/data/garden/irena/2022-10-25/renewable_energy_patents.meta.yml

```diff
@@ -0,0 +1,159 @@
+dataset:
+  title: Renewable energy patents
+  description: |
+    The data reports the annual number of new patents filed under the category of each renewable technology.
+
+    Note that the number of patents for the last few years may be subject to a time lag; processing times of patent applications vary and some patents submitted over this period may not yet be recorded in the statistics. These figures will be updated with time if additional patent applications are recorded.
+
+  sources:
+    -
+      name: International Renewable Energy Agency (IRENA) (2021)
+      published_by: International Renewable Energy Agency (IRENA) INSPIRE Platform based on EPO PATSTAT 2021 Autumn edition and on the Climate Change Mitigation Technologies (Y02) classification by EPO.
+      url: https://www.irena.org/Statistics/View-Data-by-Topic/Innovation-and-Technology/Patents-Evolution
+      date_accessed: 2022-10-25
+
+tables:
+  renewable_energy_patents:
+    variables:
+      patents:
+        title: Patents
+        short_unit: patents
+        unit: patent
+        description: Number of patents.
```

Comment:
I would add the number of decimal of places option and set it as 0 as the default in grapher is 2.

Reply:
Thanks, good suggestion!